modules/apps/export-import/export-import-web/src/main/resources/META-INF/resources/export_import_process.jsp

```diff
@@ -105,12 +105,14 @@ if (Validator.isNotNull(backURL)) {
 					long currentProgressBarCountersTotal = currentModelAdditionCountersTotal + currentPortletAdditionCounter;
 
 					if (allProgressBarCountersTotal > 0) {
-						percentage = Math.round((float)currentProgressBarCountersTotal / allProgressBarCountersTotal * 100);
+						int roundedPercentage = Math.round((float)currentProgressBarCountersTotal / allProgressBarCountersTotal * 100);
+
+						percentage = Math.max(0, Math.min(roundedPercentage, 100));
 					}
 					%>
 
 					<div class="active progress">
-						<div class="progress-bar" style="width: <%= percentage %>%;">
+						<div class="progress-bar progress-width-<%= percentage %>">
```

Comment:
Do this with a clay progress bar:

```
<clay:progressbar
  maxValue="<%= 100 %>"
  minValue="<%= 0 %>"
  value="<%= percentage %>"
/>
```

This way we can simplify the css and the logic.